code/js/content.js

```diff
@@ -289,8 +289,14 @@ function sanitize(text) {
 $(document).ready(function() {
   var $searchContainer = $('form#search_form, .site-search');
   var $q = $searchContainer.find('input[name="q"]');
-  var isRepository = $searchContainer.hasClass('repo-scope');
+  var isRepository = $searchContainer.find('.scope-badge').text() === 'This repository';
+  var isOrganization = $searchContainer.find('.scope-badge').text() === 'This organization';
 
```

Comment:
That's unfortunately the best I could think of for now to detect in which scope we are.